app/workers/delete_old_transactions_job.rb

```diff
@@ -0,0 +1,10 @@
+# frozen_string_literal: true
+
+class DeleteOldTransactionsJob
+  include Sidekiq::Worker
+  # :nocov:
+  def perform
+    AsyncTransaction::Base.delete_stale
+  end
```

Comment:
@emersonthis - re:  https://github.com/department-of-veterans-affairs/vets-api/pull/2038#discussion_r195614869

IMO, I like that you had the bang version of `delete!`.  That would ensure that errors didn't fail silently.  And then, in this job, you could rescue from that error, and log the error, and db record ID, to sentry.  Giving us insight into why things are not behaving as we expected, once this reaches prod.  

What are your thoughts on that?